src/containerapp/HISTORY.rst

```diff
@@ -10,6 +10,9 @@ upcoming
 * Upgrade api-version to 2023-11-02-preview
 * 'az containerapp create/update/up': support --build-env-vars to set environment variables for build
 * 'az containerapp create/update': support --max-inactive-revisions
+* 'az containerapp env java-component': support list Java components
```

Comment:
You also add support java component in command `az containerapp create/update`, need to add it in the history too.
`'az containerapp create/update': support bind Java component with --bind`